src/nvme/types.h

```diff
@@ -2773,6 +2773,11 @@ struct nvme_self_test_log {
 	struct nvme_st_result	result[NVME_LOG_ST_MAX_RESULTS];
 } __attribute__((packed));
 
+enum nvme_cmd_get_log_telemetry_host_lsp {
+	NVME_LOG_TELEM_HOST_LSP_RETAIN			= 0,
+	NVME_LOG_TELEM_HOST_LSP_CREATE			= 1,
+};
```

Comment:
Is this on purpose here? Looks like a left over from something else

Reply:
Oh, no, that's on purpose.
Originally these definitions were internally in srv/nvme/ioctl.c, which won't work once I've pulled the declaration into src/nvme/ioctl.h . And it arguably should have been in srv/nvme/types.h in the first place.

Reply:
ah okay, didn't spot the removal lines. the web ui stuff isn't good for this.